lib/src/kvs/rocksdb/mod.rs

```diff
@@ -22,13 +23,16 @@ pub struct Transaction {
 	rw: bool,
 	// The distributed datastore transaction
 	tx: Arc<Mutex<Option<rocksdb::Transaction<'static, OptimisticTransactionDB>>>>,
+	// the above, supposedly 'static, transaction actually points here, so keep the memory alive
+	// note that this is dropped last, as it is declared last
+	_db: Pin<Arc<rocksdb::OptimisticTransactionDB>>,
```

Comment:
Shall we just rename this property `db` as opposed to `_db`. It's private anyway...?

Reply:
I believe it is standard practice in Rust to use `_` prefix when we only care about the destructor side effect (in this case, potentially dropping the datastore if there are no more references), and don't plan on using the variable (see last point here: https://stackoverflow.com/a/48361729)

In various other languages, `_` is used for all fields of a class, but that was not the intent here.

Let me know what you think, considering the above!